### ISSUE TITLE
Error with illumate/view >= v9.36.0

### ISSUE THREAD

**Issue by mdelange** [ryangjchandler/standalone-blade#3]
Thanks a lot for providing this package! Using it refreshed my WordPress theme development 👍

This error occurs when illuminate/view >= v.9.36.0 is installed:

    Error: Call to undefined method Illuminate\Container\Container::terminating()
    (illuminate/view/ViewServiceProvider.php:26:13)

called by:

    - Illuminate\View\ViewServiceProvider:register()
      (ryangjchandler/standalone-blade/src/Blade.php:47:12)
    - RyanChandler\Blade\Blade:init()
      (ryangjchandler/standalone-blade/src/Blade.php:33:11)

illuminate/view/ViewServiceProvider.php >= v9.36.0:

```php
public function register()
{
  $this->registerFactory();
  $this->registerViewFinder();
  $this->registerBladeCompiler();
  $this->registerEngineResolver();

  $this->app->terminating(static function () {
      Component::flushCache();
  });
}
```

illuminate/view/ViewServiceProvider.php < v9.36.0

```php
public function register()
{
  $this->registerFactory();
  $this->registerViewFinder();
  $this->registerBladeCompiler();
  $this->registerEngineResolver();
}
```

**Comment by cba85** [comment]
Here's my quickfix:

I've changed `composer.json` file to let the package uses latest components versions compatible (last version before 9.36.0 for illuminate/view) :

```
"illuminate/config": "^9.0",
"illuminate/support": "^9.0",
"illuminate/view": "9.35.1"
```

To make it works in my app, I've copied this modified package in a `vendor-local` folder, and I've used the [Composer repository feature](https://getcomposer.org/doc/05-repositories.md#path) to depends on this fixed local repository instead.

Lines modified in the `composer.json` file of my app:

```
 "require": {
    "ryangjchandler/standalone-blade": "*",
  },
"repositories": [
    {
      "type": "path",
      "url": "./vendor-local/ryangjchandler/standalone-blade"
    }
  ]
```

**Comment by ryangjchandler** [comment]
Please feel free to open a PR with required changes.

**Comment by cba85** [comment]
Hello Ryan, 

First, thank you for your time spent to make this package 👍 

I am not sure my solution is a viable solution. It's just a quick fix, and that solution makes `illuminate/view` dependency fixed to a specific version (9.35.1), released on Oct 10, 2022.

The problem are `terminating()` method calls in [Illuminate/View/ViewServiceProvider.php](https://github.com/illuminate/view/blob/master/ViewServiceProvider.php) added since 9.36.0 version of the package.

I've tried to use the Decorator pattern to create a `ContainerDecorator` class and adds a `terminating()` method. It fixes only the `$this->app->terminating(...)` callings on line 126 and 166, but not the ` $app->terminating(...)` on line 55.

That's why I've decided to copy the `Illuminate/View/ViewServiceProvider.php` file inside the package, edit some calls to make it works inside the package, and remove every calls to `terminating()` method.

It works, and the package is not fixed to the 9.35.1 version of `Illuminate/View` package anymore.

I don't know if it's the best solution, but it's a better one than my first quick fix in my opinion, because the package doesn't depend to a specific `Illuminate/View` package version.

I've just open a PR if you think my solution is a good one.
